pithos/plugins/notification_icon.py

```diff
@@ -91,7 +111,18 @@ def on_has_name_owner(bus, result):
                 self.prepare_complete()
             else:
                 # No fancy tray here
-                self.prepare_complete()
+                logging.info('Checking if org.gnome.Shell is owned and version')
+                Gio.DBusProxy.new(
+                    self.bus,
+                    Gio.DBusProxyFlags.NONE,
+                    None,
+                    'org.gnome.Shell',
```

Comment:
The only real alternatives are either just skip the check and always assume one way or the other or just completely disable the notification icon plugin altogether.